modules/calypsoify/_colors.scss

```diff
@@ -1,70 +1,70 @@
+@import 'muriel-wpcom';
 // Colors that aren't pulled from Calypso or Core:
 
-$_0099d8: #0099d8; // Pacific Blue
-$_0096c3: #0096c3; // Pacific Blue
-$_008ab3: #008ab3; // Bondi Blue
-$_004967: #004967; // Prussian Blue
+$_0099d8: hsl(198, 100%, 42%); // Pacific Blue
```

Comment:
oops. i think i switched these to hsl while testing... should move back to hex or replace them entirely with more semantic names